gulp.action.sh

```diff
@@ -30,4 +30,13 @@ for i in "$@"; do
     esac
 done
 
-npx gulp "${TASK}" --platform="${PLATFORM}" --buildMode="${BUILD_MODE}"
\ No newline at end of file
+if [[ "${PLATFORM}" == "android" && "${BUILD_MODE}" == "release"  && ("${KEY_STORE_PASSWORD}" == "" || "${KEY_STORE_CONTENTS}" == "") ]]; then
```

Comment:
Don't pass KEY_STORE_CONTENTS. We should pass the file name instead.
Also, I think the names are wrong.

Perhaps we do the test in the gulp code instead?